README.md

```diff
@@ -57,9 +57,11 @@ If you're compiling your JavaScript and can't control the final `require` order,
     $ node -r newrelic your-program.js
     $ node --require newrelic your-program.js
 
+## Getting Started
+
 For more information on getting started, [check the Node.js docs](https://docs.newrelic.com/docs/agents/nodejs-agent/getting-started/introduction-new-relic-nodejs).
 
-## Using the API
```

Comment:
This section is specific to using the agent API. As such, perhaps it should be nested under "Usage" if it is important to have that as a header but I would not remove reference to the API for the content below it.

Reply:
I added a "Using the API" section under the Usage header. I think your point is super valid, but I do think it's important to keep the Usage header for consistency with other repos.